docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -0,0 +1,378 @@
+
+Next, call `add` on your instance of `proofsOnlyZkApp`:
+
+```typescript
+  const txn1 = await Mina.transaction(deployerAccount, () => {
+    proofsOnlyInstance.add(Field(4));
+  });
+
+  await txn1.prove();
+
+  await showTxn(txn1, 'txn1', legend);
+
+  await txn1.send();
+```
+
+This yields the following visualization of `txn1`
+
+
+<img src="/img/tutorial-10-txn1.png" width="100%" />
+
+See download link [here](/img/tutorial-10-txn1.png).
+
+Now there are two AccountUpdates, where one is a child of the other. The parent corresponds to the `add` method call - and the child corresponds to the `this.incrementCalls()` call that the parent makes. 
+
+One update is the child because it was called from the parent - which also implies that the parent has the child included as part of its proof. You can read more on parent / child account updates [here](../how-to-write-a-zkapp#payments-and-more-on-public-inputs).
+
+As a reminder, this corresponds to code: 
+
+```typescript
+  ...
+  @method add(incrementBy: Field) {
+    this.account.provedState.assertEquals(this.account.provedState.get());
+    this.account.provedState.get().assertTrue();
+
+    const num = this.num.get();
+    this.num.assertEquals(num);
+    this.num.set(num.add(incrementBy));
+
+    this.incrementCalls();
+  }
+
+  @method incrementCalls() {
+    this.account.provedState.assertEquals(this.account.provedState.get());
+    this.account.provedState.get().assertTrue();
+
+    const calls = this.calls.get();
+    this.calls.assertEquals(calls);
+    this.calls.set(calls.add(Field(1)));
+  }
+  ...
+```
+
+You can note in the account updates that in the first update, you're updating the state of `appState[0]` to `5` (matching the `this.num.set` in the `add` contract), and in the second update, you're updating `appState[1]` to `1` (matching the `this.calls.set` in the `incrementCalls` contract).
+
+Lastly, call `callSecondary` on your instance of `proofsOnlyZkApp`:
+
+```typescript
+  const txn2 = await Mina.transaction(deployerAccount, () => {
+    proofsOnlyInstance.callSecondary(secondaryAddr);
+  });
+
+  await txn2.prove();
+
+  await showTxn(txn2, 'txn2', legend);
+  await saveTxn(deploy_txn, 'deploy_txn', legend, './txn2.png');
+
+  await txn2.send();
+```
+
+This yields the following visualization of `txn2`
+
+<img src="/img/tutorial-10-txn2.png" width="100%" />
+
+See download link [here](/img/tutorial-10-txn2.png).
+
+And a quick reminder of the code for `callSecondary`:
+
+```typescript
+  @method callSecondary(secondaryAddr: PublicKey) {
+    this.account.provedState.assertEquals(this.account.provedState.get());
+    this.account.provedState.get().assertTrue();
+
+    const secondaryContract = new SecondaryZkApp(secondaryAddr);
+
+    const num = this.num.get();
+    this.num.assertEquals(num);
+
+    secondaryContract.add(num);
+
+    // NOTE this gets the state at the start of the transaction
```

Comment:
@barriebyron maybe we should establish a style for this. Probably not all caps

Reply:
No guidelines exist for the style of single line code comments. `//` is ideal for the syntax for a single-line JS comment, but not sure about style of the prefix there.

For methods and classes in our code, we use [JS Doc](https://jsdoc.app) comments as our guide, but that won't help here. We haven't defined to greater specificity than that.